lib/dhcp/client/client.go

```diff
@@ -85,6 +89,12 @@ func (c *client) SetTimeout(t time.Duration) {
 	c.timeout = t
 }
 
+func (c *client) SetHostName(hostname string) {
+	defer trace.End(trace.Begin(""))
```

Comment:
It looks not worth tracing this simple set method.

Reply:
Done